index.yaml

```diff
@@ -112,6 +112,12 @@ indexes:
   - name: exploration_id
   - name: last_updated
 
+- kind: GeneralSuggestionModel
+  properties:
+  - name: deleted
```

Comment:
not sure why "deleted" is in here? If you remove this stanza and rerun on dev server, does this get autogenerated exactly the way it is now?

Reply:
This was auto generated. Will try again and let you know.

Reply:
Just confirmed. So I deleted these lines, and ran the cron job and these lines got added back. Looking around this file a few of the other models also had "deleted". But I have no idea on why "deleted" is being added here.

Reply:
Hm OK, no idea either. But happy to merge! Thanks for confirming.

Reply:
Thanks!